system/jlib/jstats.cpp

```diff
@@ -396,6 +400,43 @@ static StringBuffer & formatIPV4(StringBuffer & out, unsigned __int64 value)
     return out.appendf("%d.%d.%d.%d", ip1, ip2, ip3, ip4);
 }
 
+static const char *getMoneyLocale()
+{
+    static bool haveLocale = false;
+    static StringAttr moneyLocale;
+
+    if (!haveLocale)
+    {
+        StringBuffer localestr;
+        if (isGlobalConfigReady())
```

Comment:
Why is this check necessary?  The global configuration should be set up by all components early in their lifetime.